content/learn/quick-start/getting-started/setup.md

```diff
@@ -143,84 +143,118 @@ You might think to simply develop in release mode instead, but we recommend agai
 
 ### Enable Fast Compiles (Optional)
 
-Bevy can be built just fine using default configuration on stable Rust. However for maximally fast iterative compiles, we recommend the following configuration:
+Bevy can be built just fine using default configuration on stable Rust.
+Unfortunately, the compile times are rather long.
+This section explains how to improve iterative compile times.
```

Comment:
```suggestion
This section explains how to speed up iterative compiles: the amount of time it takes to rebuild your project after changing a single file.
```

Reply:
Perfect; I knew this needed a something else but couldn't quite find it